integrated_channels/xapi/models.py

```diff
@@ -39,6 +48,25 @@ class XAPILRSConfiguration(TimeStampedModel):
         null=False,
         help_text=_('Is this configuration active?'),
     )
+    auth_method = models.CharField(
+        max_length=16,
+        verbose_name="xAPI POST Authentication Method",
+        choices=XAPIAuthMethods.choices,
+        default=XAPIAuthMethods.HTTP_BASIC,
+        help_text=_('The Authentication Method to use when sending the xAPI data to the endpoint.')
+    )
+    auth_url = models.URLField(
+        blank=True,
+        null=True,
+        help_text=_("URL to use for authentication. Eg., Token URL for OAuth")
+    )
+    oauth_scope = models.CharField(
+        max_length=255,
+        verbose_name=_('OAuth scope'),
+        blank=True,
+        null=True,
+        help_text=_('The "scope" to pass for OAuth authentication.')
```

Comment:
What scopes are available here? Could we list them?

Reply:
This would be specified by the LRS as well.